prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres']
+console.log(languages);
+
+const firstElement = languages [0];
+console.log(firstElement);
+
+const thirdElement = languages [2];
+console.log(thirdElement);
+
+const length = languages.length;
+console.log(length);
+
+const lastElement =languages [5];
```

Comment:
this element should be accessed by calculating the end of the array using the length variable created on line 10. Please reach out if you have any questions!